src/javascripts/components/myNavbar/myNavBar.js

```diff
@@ -1,27 +1,31 @@
+import firebase from 'firebase/app';
+import 'firebase/auth';
+
 import utils from '../../helpers/utils';
 
+const logoutEvent = () => {
+  $('.navbar-logout-button').click((e) => {
+    e.preventDefault();
+    firebase.auth().signOut();
+    $('.navbar-login-button').removeClass('hide');
```

Comment:
navbar-logout-button will be changed to an id so this should become `#navbar-logout-button`

Reply:
done